packages/site/formideploy.config.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Formideploy configuration overrides.
+ */
+module.exports = {
+  lander: {
+    name: 'urql',
+  },
+  production: {
+    // TODO: temp retarget when we have verified this working
+    domain: 'tmp-experiment-02.formidable.com',
+    bucket: 'tmp-experiment-02.formidable.com',
+  },
+};
```

Comment:
Why is this a random `.config.js` file? Would this be more suited in a different location? And I think it should be safe to use the prod bucket here, assuming it's the same logic as before, since the scripts seem almost identical :+1: